### ISSUE TITLE
Numbers and Booleans don't display in green in cells (3.2 beta)

### ISSUE THREAD

**Issue by ostephens** [OpenRefine/OpenRefine#1996]
**Describe the bug**
When a cell is populated with a number or boolean, it doesn't not display as green text in the UI

**To Reproduce**
1. Open a project
2. Edit a cell to a number or boolean
3. View project

**Current Results**
Cell text appears in the same colour as a string

**Expected behavior**
Cell text for numbers and booleans should display in green
(n.b. this behaviour works for Dates)

**Desktop (please complete the following information):**
 - OS: macOS Mojave
 - Browser Version: Safari 12.0.3

**OpenRefine (please complete the following information):**
 - 3.2 Beta

**Comment by ostephens** [comment]
I think this might be part of a more serious bug where types aren’t being applied correctly on the backend. More investigation needed

**Comment by ostephens** [comment]
JSON being returned on a get-rows request isn't include a "t" property for numbers/booleans

```
{
    "mode": "row-based",
    "rows": [
        {
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": "2019-01-01T00:00:00Z",
                    "t": "date"
                }
            ],
            "i": 0
        },
        {
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": "1.0"
                }
            ],
            "i": 1
        },
        {
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": "2.0"
                }
            ],
            "i": 2
        },
        {
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": "true"
                }
            ],
            "i": 3
        }
    ],
    "filtered": 4,
    "total": 4,
    "start": 0,
    "limit": 4,
    "pool": {
        "recons": {}
    }
}
```

The second row should have `"t": "number"`

Might be related to Jackson migration and serialisation of the JSON?

**Comment by wetneb** [comment]
Probably! Thanks for the debugging, I'll look into that.

**Comment by ostephens** [comment]
OK - I was wrong about that - the issue is not the lack of a "t" value but that in the JSON the number and boolean is being included as a string not a JSON number/boolean. So instead of:
```
{
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": "true"
                }
            ],
            "i": 3
        }
```
it should be
```
{
            "starred": false,
            "flagged": false,
            "cells": [
                {
                    "v": true
                }
            ],
            "i": 3
        }
```

**Comment by ostephens** [comment]
Closed by #2008
